### PR TITLE
Deployment param update

### PR DESCRIPTION
**Pull request by brimigs**
Add in Mars Hub Addresses to Address provider (needed for rewards collector contract to send rewards)

### CODE REVIEW

File: scripts/deploy/base/deployer.ts [hunk 1]
```diff
@@ -182,6 +182,18 @@ export class Deployer {
...
+      {
+        address_type: 'fee_collector',
+        address: this.config.feeCollectorAddr,
+      },
+      {
+        address_type: 'safety_fund',
+        address: this.config.safetyFundAddr,
+      },
+      {
+        address_type: 'protocol_admin',
```
Comment (piobab) [inline scripts/deploy/base/deployer.ts:194]: You can remove `protocol_admin` above `red_bank`

File: scripts/deploy/osmosis/config.ts [hunk 2]
```diff
@@ -116,6 +116,10 @@ export const osmosisMainnet: DeploymentConfig = {
       ],
     },
   ],
+  // note the following addresses are all 'mars' bech32 prefix
+  safetyFundAddr: 'TBD',
```
[the remark this answers is not in the document]
Comment (brimigs) [reply scripts/deploy/osmosis/config.ts:120]: these are testnet addresses?

[the remark this answers is not in the document]
Comment (piobab) [reply scripts/deploy/osmosis/config.ts:120]: Yes but Larry said that they will be the same for mainnet

[the remark this answers is not in the document]
Comment (larry0x) [reply scripts/deploy/osmosis/config.ts:120]: @piobab `protocol_admin` should initially be the Osmosis dev multisig, and later when the envoy module on Mars Hub is ready, transferred to the interchain account.

[the remark this answers is not in the document]
Comment (larry0x) [reply scripts/deploy/osmosis/config.ts:120]: For `fee_collector` and `safety_fund`, their addresses are derived deterministically from the module names (`"fee_collector"` and `"safety"`), so they are the same for testnet & mainnet.

**Review by larry0x — APPROVED** [review]
Can be merged once the 3 addresses (`safetyFundAddr`, `protocolAdminAddr`, `feeCollectorAddr`) are added

File: scripts/deploy/osmosis/config.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (brimigs) [reply scripts/deploy/osmosis/config.ts:120]: added them

File: scripts/deploy/base/deployer.ts (hunk 1 above)
Comment (brimigs) [reply scripts/deploy/base/deployer.ts:194]: updated

File: scripts/deploy/osmosis/config.ts [hunk 3]
```diff
@@ -33,6 +33,10 @@ export const osmosisTestnetConfig: DeploymentConfig = {
   swapRoutes: [
     { denom_in: atom, denom_out: 'uosmo', route: [{ pool_id: 1, token_out_denom: 'uosmo' }] },
   ],
+  // note the following addresses are all 'mars' bech32 prefix
+  safetyFundAddr: 'mars1s4hgh56can3e33e0zqpnjxh0t5wdf7u3pze575',
```
Comment (piobab) [inline scripts/deploy/osmosis/config.ts:37]: Could you please create variable with common values (safetyFundAddr, protocolAdminAddr, feeCollectorAddr) and reuse, such as:
```
const axlUSDC = 'ibc/D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858'
const atom = 'ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2'
const marsTestnet = 'ibc/ACA4C8A815A053CC027DB90D15915ADA31939FA331CE745862CDD00A2904FA17'
```

**Review by piobab — APPROVED** [review]

**Pull request merged**